app/views/legal/index.html.erb

```diff
@@ -56,7 +56,7 @@
   </div>
   </span>
     <span class="flex w-full items-center pl-16 pr-16">
-      <%= image_tag 'tribo_logo.svg' %>
+      <%= image_tag 'tribo-logo.svg' %>
```

Comment:
I think this fix will cause a problem with PR feature # 27, since as discussed in the board, everything was moved from legal/index to market_places/index

Reply:
At the moment I made this change and opened the PR feature #27 was still not merged, so let me fix this, because I change it to see if the logo was working or not

Reply:
It only remains to fix the icon, so that it does not cause this problem.

![imagen](https://user-images.githubusercontent.com/12243331/104750661-046dc100-571a-11eb-92af-0387ab979ce4.png)